### PR TITLE
challenge 1 completed

### PR DESCRIPTION
**Pull request by enaburekhan**

### CODE REVIEW

File: samplescalaproject/src/main/scala/example/Hello.scala [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package example
+
+object Hello extends Greeting with App {
+  println("Welcome to 30 days of Scala")
+}
+
+trait Greeting {
+  lazy val greeting: String = "hello"
```
Comment (ayojava) [inline samplescalaproject/src/main/scala/example/Hello.scala:8]: the changes are supposed to be made on the greeting variable 
```suggestion
  lazy val greeting: String = "Welcome to 30 days of Scala"
```

File: samplescalaproject/src/main/scala/example/Hello.scala [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package example
+
+object Hello extends Greeting with App {
+  println("Welcome to 30 days of Scala")
```
Comment (ayojava) [inline samplescalaproject/src/main/scala/example/Hello.scala:4]: You will then reference the "greeting" 
```suggestion
  println(greeting)
```

**Review by ayojava — COMMENTED** [review]
Requested some changes to be done . If you have questions , you can drop it on the channel

File: samplescalaproject/src/main/scala/example/Hello.scala (hunk 2 above)
Comment (enaburekhan) [reply samplescalaproject/src/main/scala/example/Hello.scala:4]: ok thanks for the observation. I will the changes and revert back

Comment (enaburekhan) [reply samplescalaproject/src/main/scala/example/Hello.scala:4]: *make

**Pull request closed without merging**